docs/installation-guide/quickstart.rst

```diff
@@ -0,0 +1,345 @@
+to installation procedures, the only significant difference between a test
+cluster and a full production environment is the amount of resources required
+to implement the design.
+
+.. _MetalK8s: https://github.com/scality/metalk8s/
+.. _CentOS: https://www.centos.org
+.. _Kubernetes: https://kubernetes.io
+
+General Cluster Requirements
+----------------------------
+Setting up a MetalK8s_ cluster quickly requires at least three machines
+running CentOS_ 7.6 or higher (these can be VMs) to which you have SSH access.
+Each machine acting as a Kubernetes_ node (all three in the present example)
+must also have at least one disk available to provision storage volumes.
+Since the version 2.x of MetalK8s_ you need to install first a bootstrap node
+and then the others nodes (2 in our case).
+
+Sizing
+^^^^^^
+
+Each node must satisfy the following sizing requirements.
+
+.. note::
+   The root file system requires at least 40 GB.
+
++-----------------+--------+--------+-------+
+|    Component    | etcd   | Master | Node  |
++=================+========+========+=======+
+| Cores           | 2      | 4      | 4     |
++-----------------+--------+--------+-------+
+| RAM             | 4 GB   | 8 GB   | 8 GB  |
++-----------------+--------+--------+-------+
+| Minimum         |        |        |       |
+| dedicated       |        |        |       |
+| storage capacity|        |        |       |
+| required        |    -   |    -   | 128 GB|
++-----------------+--------+--------+-------+
+
+Proxies
+^^^^^^^
+For nodes operating behind a proxy, add the following lines to each cluster
+server's :file:`/etc/environment` file:
+
+.. code-block:: shell
+
+ http_proxy=http://user;pass@<HTTP proxy IP address>:<port>
+ https_proxy=http://user;pass@<HTTPS proxy IP address>:<port>
+ no_proxy=localhost,127.0.0.1,<local IP of each node>
+
+Mount MetalK8s ISO
+-------------------
+Get the ISO file and mount it on the bootstrap node at this specific path.
+
+  .. code-block:: shell
+
+   $ mkdir -p /srv/scality/metalk8s-2.0
+   $ mount <path-to-iso> /srv/scality/metalk8s-2.0
+
+
+Prepare the bootstap node
+-------------------------
+1. Create the :file:`/etc/metalk8s/bootstrap.yaml` file. Change the netmask,
+   IP addresse, and name to conform to your infrastructure.
+
+   .. code-block:: yaml
+
+    apiVersion: metalk8s.scality.com/v1alpha2
+    kind: BootstrapConfiguration
+    networks:
+     controlPlane: <netmask>/24
+     workloadPlane: <netmask>/24
+    ca:
+     minion: <name-of-the-bootstrap-node>
+    apiServer:
+     host: <IP-of-the-bootstrap-node>
+
+2. Generate the ssh key of the bootstrap and copy it.
+
+   .. code-block:: shell
+
+    $ ssh-keygen
+    $ ssh-copy-id <IP-of-the-bootstrap-node>
+
+3. Copy the private key in the pki folder of MetalK8s. This will be use further for
+   adding new nodes on the cluster.
+
+   .. code-block:: shell
+
+    $ mkdir -p /etc/metalk8s/pki/
+    $ cp /root/.ssh/id_rsa /etc/metalk8s/pki/id_rsa
```

Comment:
Is it standard procedure? Maybe generating a new key for this specific purpose is better practice, I'm not certain.

Reply:
See #1122